src/OrderBy/OrderByDirective.php

```diff
@@ -39,6 +40,22 @@ public static function definition(): string
     """
     columnsEnum: String
 ) on ARGUMENT_DEFINITION
+
+"""
+Sort a result list by a column
+"""
+directive @orderBy(
```

Comment:
That will lead to errors in the generated `schema-directives.graphql`, directives names must be unique. You can just merge the two definitions and clarify correct usage in the description.

Reply:
Oh, I thought they just had to be unique based on where they were applied.